pygfx/resources/_texture.py

```diff
@@ -33,6 +33,11 @@ def __init__(self, data=None, *, dim, size=None, format=None):
         self._data = None
         self._pending_uploads = []  # list of (offset, size) tuples
 
+        assert encoding in ("linear", "srgb")
```

Comment:
```suggestion
        if encoding not in ("linear, "srgb"):
            raise ValueError("encoding must be one of 'linear' and 'srgb'")
```

Reply:
Nah, this will be replaced with the one from the gamma PR anyway.